setup.py

```diff
@@ -1,12 +1,8 @@
-import sys
-
 from setuptools import setup
 
 packages = ["raygun4py", "raygun4py.middleware"]
 
-base_dir = "python2"
-if sys.version_info[0] == 3:
-    base_dir = "python3"
```

Comment:
Do we still support python 2?

Reply:
We don't support it in the project anymore, the code was deleted back in version 5, so having this setup step would be incorrect. 

Users can still use version 4.4.0 if they want to use python2